Intersect.Server.Core/Entities/Player.cs

```diff
@@ -1395,19 +1395,19 @@ public void TakeExperience(long amount, bool enableLosingLevels = false, bool fo
             }
             else
             {
-                var levelCount = 0;
-                while (Exp < 0 && Level + levelCount > 1)
+                var levelsToRemove = 0;
+                while (Exp < 0 && Level - levelsToRemove > 1)
                 {
-                    --levelCount;
-                    Exp += GetExperienceToNextLevel(Level + levelCount);
+                    ++levelsToRemove;
+                    Exp += GetExperienceToNextLevel(Level - levelsToRemove);
                 }
 
                 if (Exp < 0)
                 {
                     Exp = 0;
                 }
 
-                AddLevels(levelCount); //AddLevels includes PacketSender.SendExperience(this);
+                AddLevels(-levelsToRemove); //AddLevels includes PacketSender.SendExperience(this);
```

Comment:
I wrote the code as you wrote it now, at the beginning, but panda told me to revert it to how it is currently, so he won't approve this change